cmd/minishift/cmd/completion.go

```diff
@@ -31,15 +31,20 @@ const longDescription = `
 	Outputs minishift shell completion for the given shell (bash or zsh)
 
 	This depends on the bash-completion binary.  Example installation instructions:
+
 	OS X:
```

Comment:
This should refer to macOS instead of OS X.

Reply:
@robin-owen sure, I was expecting this comment.